ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/KsqlRequest.java

```diff
@@ -46,7 +47,7 @@ public KsqlRequest(
     this.ksql = ksql == null ? "" : ksql;
     this.streamsProperties = streamsProperties == null
         ? Collections.emptyMap()
-        : Collections.unmodifiableMap(new HashMap<>(streamsProperties));
+        : Collections.unmodifiableMap(new HashMap<>(serializeClassValues(streamsProperties)));
```

Comment:
It would be good to add a test for this.

Reply:
Done